src/lib/stellar/transactions.js

```diff
@@ -331,3 +332,58 @@ export async function createPathPaymentStrictReceiveTransaction({
         network_passphrase: networkPassphrase,
     }
 }
+
+/**
+ * Constructs and returns a Stellar transaction for transferring assets to a contract or account.
```

Comment:
Since this application is intended to be a demonstrative resource, I think we should keep this function limited in scope to _only_ performing SAC transfers. We have the other `payment` operation transaction already being handled by another function. Let's make this one focused on SAC transfers, solely.